contractcourt/briefcase.go

```diff
@@ -1475,12 +1475,18 @@ func decodeBreachResolution(r io.Reader, b *BreachResolution) error {
 	return binary.Read(r, endian, &b.FundingOutPoint.Index)
 }
 
-func encodeHtlcSetKey(w io.Writer, h *HtlcSetKey) error {
-	err := binary.Write(w, endian, h.IsRemote)
+func encodeHtlcSetKey(w io.Writer, h fn.Option[HtlcSetKey]) error {
```

Comment:
I wouldn't bother having this take an option. Just have it take the thing and put this function in a `WhenSome` or something like that.